lib/backend/syncersv1/felixsyncer/felixsyncer_e2e_test.go

```diff
@@ -70,36 +70,36 @@ var _ = testutils.E2eDatastoreDescribe("Felix syncer tests", testutils.Datastore
 				syncTester.ExpectData(model.KVPair{
 					Key: model.ProfileRulesKey{ProfileKey: model.ProfileKey{Name: "kns.default"}},
 					Value: &model.ProfileRules{
-						InboundRules:  []model.Rule{{Action: "allow"}},
-						OutboundRules: []model.Rule{{Action: "allow"}},
+						InboundRules:  []model.Rule{{Action: "Allow"}},
```

Comment:
Everything in model (apart from the resources key and list) are basically the v1 model.  Therefore the values in these structs should be the v1 values (i.e. lower case)

Reply:
should be fixed in recent commit.

Reply:
still seeing uppercase here.